tarantool/connection.py

```diff
@@ -250,17 +251,18 @@ def _read_response(self):
 
     def _send_request_wo_reconnect(self, request):
         '''
-        :rtype: `Response` instance
+        :rtype: `Response` instance or subclass
 
         :raise: NetworkError
         '''
-        assert isinstance(request, Request)
+        if not issubclass(type(request), Request):
+            raise NetworkError
```

Comment:
I rolled back this change because it is not related with dbapi module and must be done in a separate refactoring commit which will affect all the similar places across the package modules.